charts/timed/templates/ingress.yaml

```
@@ -12,10 +12,10 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- with .Values.ingress.annotations }}
   annotations:
-    {{- range $key, $value := .Values.ingress.annotations }}
-      {{ $key }}: {{ $value | quote }}
-    {{- end }}
+{{- toYaml . | nindent 4 }}
```

Either indent the nindent or use a simple indent?
```suggestion
    {{- toYaml . | nindent 4 }}
```